app/models/simple_smart_answer_edition.rb

```diff
@@ -80,6 +80,26 @@ def destroy_in_attrs?(attrs)
     attrs.delete("_destroy") == "1"
   end
 
+  def generate_mermaid
+    parts = ["%%{ init: {\n'theme': 'base',\n'themeVariables': {\n    " \
+      "'background': '#FFFFFF',\n    'primaryTextColor': '#0B0C0C',\n    " \
+      "'lineColor': '#0b0c0c',\n    'fontSize': '23.75px' } } }%%\nflowchart TD"]
+    parts << "accTitle: #{title}\naccDescr: A flowchart for the #{title} smart answer\nAA[Start]:::start"
+    unless nodes.nil?
+      parts << "AA---Q#{nodes.first.slug.split('-')[1]}"
```

Comment:
I can't find exactly where slugs are set in the code, but it can't be done manually and that's how they look in the db. i would expect it maybe to be set in the migrations? but i cant find the migrations